app/src/ui/release-notes/release-notes-dialog.tsx

```diff
@@ -82,11 +82,47 @@ export class ReleaseNotes extends React.Component<IReleaseNotesProps, {}> {
     )
   }
 
+  /**
+   * If there is just one release, it returns it. If multiple, it merges the release notes.
+   */
+  private getDisplayRelease = () => {
+    if (this.props.newReleases.length === 1) {
+      return this.props.newReleases[0]
+    }
+
+    const earliestRelease = this.props.newReleases.slice(-1)[0]
```

Comment:
Shouldn't this be `oldestRelease`? While I guess an older release happened earlier than a later release it confused the heck out of me since we're comparing with `latestRelease` below.

Reply:
:D - To me it is relays the same meaning so happy to change it.

Reply:
Thank you, it's probably just a language barrier thing but `oldest` makes my brain happy!